packages/auth/demo/src/index.js

```diff
@@ -114,7 +114,7 @@ const providersIcons = {
 };
 
 /**
- * Returns the active user (i.e. currentUser or lastUser).
+ * Returns active user (i.e. currentUser or lastUser).
```

Comment:
Suggest "Returns the active user (`currentUser` or `lastUser`)."

Always avoid "i. e."